### DISCUSSION TITLE
articles/wechat-mini-program-sourcemap/

### DISCUSSION THREAD

**Discussion by wind-stone** [wind-stone/blog#305] [Announcements]
https://blog.windstone.cc/articles/wechat-mini-program-sourcemap/ 

 记录工作，记录生活

**Comment by jerryjiao** [comment]
这个链接404了呃

**Comment by Lgowen** [comment]
你好想请教一下这个上传的sourcemap源码包指的是微信小程序平台上下载的吗，这个源码包在上传的时候是需要包含uniapp编译后的map文件也在里面对吗

**Comment by wind-stone** [comment]
> 你好想请教一下这个上传的sourcemap源码包指的是微信小程序平台上下载的吗，这个源码包在上传的时候是需要包含uniapp编译后的map文件也在里面对吗

是的，需要将 uniapp 编译后的 map 也上传到微信小程序平台，他们再基于你的上传的 map 生成新的 map 文件 @Lgowen
